test/Microsoft.ServiceFabric.Actors.IntegrationTests/Reminders/ReminderRegistrationTest.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Fabric;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceFabric.Actors.Runtime;
+using Microsoft.ServiceFabric.Services.Runtime;
+using Xunit;
+using Fuzzy;
+
+namespace Microsoft.ServiceFabric.Actors
+{
+    public class ActorIntegrationTest
+    {
+        protected async Task<ActorService> GetActorService<T>(
+            Func<ActorService, ActorId, ActorBase> actorFactory = null,
+            ActorServiceSettings actorServiceSettings = null)
+            where T : Actor
+        {
+            IFuzz fuzzy = new RandomFuzz();
+
+            ActorService actorService = new ActorService(
+                fuzzy.StatefulServiceContext(),
+                ActorTypeInformation.Get(typeof(T)),
+                actorFactory,
+                null,
+                new NullActorStateProvider(),
+                actorServiceSettings);
+
+            IStatefulUserServiceReplica statefulServiceReplica = actorService;
+            await statefulServiceReplica.OnOpenAsync(ReplicaOpenMode.New, CancellationToken.None);
+            await statefulServiceReplica.OnChangeRoleAsync(ReplicaRole.Primary, CancellationToken.None);
+            await statefulServiceReplica.RunAsync(CancellationToken.None);
+
+            return actorService;
+        }
+
+        interface ITestableActor : IActor
```

Comment:
Do we have to have the ITestableActor interface? Can we cast directly to TestableActor instead?